FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -237,6 +246,110 @@ - (void)callListeners:(NSString *)key config:(NSDictionary *)config {
   }
 }
 
+- (void)setCustomSignals:(nullable NSDictionary<NSString *, NSObject *> *)customSignals
```

Comment:
just curious, was it not feasible to accept `CustomSignalValue` for the Objective-C implementation? i remember this being a discussion topic a while ago

Reply:
Yes, on discussion with Nick on the iOS SDK team, we decided not use  CustomSignalValue for the Objective-C implementation: [Link to discussion](https://docs.google.com/document/d/1zTN6xNx2fKWONKPOy-x0nfED043jb_3KbttzjQZYq_E/edit?disco=AAABYZlUm0o)